LICENSE.md

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017 Bayes Impact
+   Copyright 2017 Beacon Labs
```

Comment:
Should this be 2018?

Reply:
this was actually auto-generated by github when we added a license. I maybe shouldn't update it at all. Not sure how licenses work really though

Reply:
went with actually switching this back to bayes impact because beacon labs isn't a legal entity yet. probably doesn't matter, but just to be safe. added it to the followup ticket